initrd/bin/flash.sh

```diff
@@ -6,15 +6,9 @@ set -e -o pipefail
 . /etc/functions
 . /tmp/config
 
-case "$CONFIG_BOARD" in
-  librem* )
-    FLASHROM_OPTIONS='-p internal:laptop=force_I_want_a_brick,ich_spi_mode=hwseq' 
-  ;;
-  x230* )
-    FLASHROM_OPTIONS='--force --noverify-all --programmer internal --ifd --image bios'
-  ;;
-  "kgpe-d16" )
-    FLASHROM_OPTIONS='--force --noverify --programmer internal'
+case "$FLASHROM_OPTIONS" in
+  -* )
```

Comment:
I have it check for -*, as  the definitions are arguments, thus they should begin with a "-."

Feel free to suggest a better way to check for valid definitions.